### PR TITLE
Add animate-initial-selection feature

### PR DESCRIPTION
**Pull request by zerodevx**
Hi @morethanreal,

Referencing #8 and #9, I've renamed the `animateInitialSelection` feature into a separate branch for your review. 

Thanks,
Jason

### CODE REVIEW

File: neon-animated-pages.html [hunk 1]
```diff
@@ -56,6 +56,8 @@
 
     is: 'neon-animated-pages',
 
+    created: function () { console.log("created"); },
```
Comment (morethanreal) [inline neon-animated-pages.html]: Remove this?

**Comment by morethanreal** [comment]
Just a comment and it seems like this needs to be rebased.

File: neon-animated-pages.html (hunk 1 above)
Comment (zerodevx) [inline neon-animated-pages.html]: Ooops yes I must be sleeping! Was doing some checks and left that in by accident.

**Comment by zerodevx** [comment]
Will do, I'll rebase, clean up and push the commit shortly. Thanks!

**Comment by zerodevx** [comment]
@morethanreal, is this ok?

**Comment by morethanreal** [comment]
lgtm

**Pull request merged**